src/test/bind/CMakeLists.txt

```diff
@@ -5,5 +5,11 @@ add_executable(test-bind test_bind.c ../test_common.c)
 
 ## register the tests
 add_test(NAME bind COMMAND test-bind)
-add_test(NAME bind-shadow-ptrace COMMAND ${CMAKE_BINARY_DIR}/src/main/shadow --interpose-method=ptrace -l debug -d bind.shadow-ptrace.data ${CMAKE_CURRENT_SOURCE_DIR}/bind.test.shadow.config.xml)
-add_test(NAME bind-shadow-preload COMMAND ${CMAKE_BINARY_DIR}/src/main/shadow --interpose-method=preload -l debug -d bind.shadow-preload.data ${CMAKE_CURRENT_SOURCE_DIR}/bind.test.shadow.config.xml)
+add_test(
+        NAME bind-shadow-ptrace
+        COMMAND sh -c "python3 ${CMAKE_SOURCE_DIR}/src/tools/convert.py yaml2xml ${CMAKE_CURRENT_SOURCE_DIR}/bind.test.shadow.config.yaml --output - | ${CMAKE_BINARY_DIR}/src/main/shadow --interpose-method=ptrace -l debug -d bind.shadow-ptrace.data -"
```

Comment:
I think breaking the command (and the one below) over two lines would help readability here. (Most of our tests currently use only a single line because we just recently upgraded to a CMake version that supports splitting them across lines.)